lib/geoengineer/gps/meta_node.rb

```diff
@@ -0,0 +1,31 @@
+require_relative "./node"
+# A MetaNode is expanded into other nodes.
+# The goal is that this can simplify the combination and reuse of nodes
+# e.g. a serivce node, and a load balancer node could be combined into a
+# service_with_load_balancer meta node
+class GeoEngineer::GPS::MetaNode < GeoEngineer::GPS::Node
+  def self.define_resource(type, name, id_lambda)
+    raise "Meta nodes cannot define resources"
+  end
+
+  def self.meta?
```

Comment:
why does it need meta at the class level?

Reply:
As a means of checking the class is a meta for various switching.